### PR TITLE
Zeta-Prebid

### PR DESCRIPTION
**Pull request by mwehr-zeta**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

A bidder adapter for Zeta Global

- test parameters for validating bids
```
{
  mediaTypes: {
    banner: {
      sizes: [[300, 250]],
    }
  },
  bids: [{
    bidder: 'Zeta Global',
    bidId: 12345,
    params: {
      placement: 12345,
      user: {
        uid: 12345,
        buyeruid: 12345
      },
      ip: '111.222.33.44',
      test: 1
    }
  }]
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: DL-ZetaDSP-Supply-Engineering@zetaglobal.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by smenzer** [comment]
please remove the `package-lock.json` file from this PR. thanks!

**Comment by mwehr-zeta** [comment]
Comments addressed and docs submitted: https://github.com/prebid/prebid.github.io/pull/2409

File: modules/zetaBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        var adUnits = [
+            {
+                mediaTypes: {
+                    banner: {
+                        sizes: [[300, 250]],  // a display size
+                    }
+                },
+                bids: [
+                    {
+                        bidder: 'Zeta Global',
```
Comment (GLStephen) [inline modules/zetaBidAdapter.md:24]: The server request failed when I submitted these test bids.

Comment (mwehr-zeta) [reply modules/zetaBidAdapter.md:24]: Both issues should be addressed

File: modules/zetaBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
+import * as utils from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+const BIDDER_CODE = 'Zeta Global';
```
[the remark this answers is not in the document]
Comment (mwehr-zeta) [reply modules/zetaBidAdapter.js]: Updated

**Comment by mwehr-zeta** [comment]
Is there an update on this?  All comments should be addressed.

**Review by GLStephen — APPROVED** [review]

**Comment by GLStephen** [comment]
@mwehr-zeta I still see one final issue, I'm just not getting any test bids back and all of the XMLHttpRequests to your endpoint are failing.

image

**Comment by ilhelka** [comment]
Hi @GLStephen,  thanks for the heads up. Responding to confirm receipt and @mwehr-zeta is addressing the issue. We should have an update soon. Cheers, Ilham E (Zeta DSP PM)

**Comment by mwehr-zeta** [comment]
We will need to complete a deployment of our bidders to address the header issue.

**Comment by GLStephen** [comment]
@mwehr-zeta please ping me when it's done and I'll recheck

**Comment by mwehr-zeta** [comment]
@GLStephen It should be good to go

**Pull request merged**